numpy/core/memmap.py

```diff
@@ -335,3 +341,22 @@ def __getitem__(self, index):
         if type(res) is memmap and res._mmap is None:
             return res.view(type=ndarray)
         return res
+
+    def close(self):
+        """
+        Close the memmap file and reduce reference count by 1.
+        If the reference count is zero, the file is unlocked.
+        
+        It has no effect if no file is memory-mapped.
+        For further information, see `memmap`.
+        
+        Parameters
+        ----------
+        None
+        See Also
```

Comment:
```suggestion

        See Also
```
As was mentioned previously, an empty line is missing here as well.